src/env.d.ts

```diff
@@ -15,18 +15,4 @@ declare module 'vue/types/vue' {
     _watchers: VueWatcher[]
     __composition_api_state__?: VfaState
   }
-
-  interface VueConstructor {
-    observable<T>(x: any): T
-    util: {
-      warn(msg: string, vm?: Vue | null)
-      defineReactive(
```

Comment:
Isn't there a way to keep this code without changing the public api on vuejs/vue code?

Reply:
@posva Maybe we could use `// @ts-ignore` or use `any` to define the type to avoid the warnings.